tests/market/test_matching_engine_external_matcher.py

```diff
@@ -43,19 +45,20 @@ def _populate_market_bids_offers(self):
     def test_subscribes_to_redis_channels(self):
         self.matcher.matching_engine_ext_conn.sub_to_multiple_channels.assert_called_once_with(
             {
-                "external-matching-engine/simulation-id/": self.matcher.publish_simulation_id,
-                f"{self.channel_prefix}offers-bids/":
+                MatchingEngineChannels(gsy_e.constants.CONFIGURATION_ID).simulation_id:
```

Comment:
Minor, but you can construct the MatchingEngineChannels object once and reuse it.